client/lxd_server.go

```diff
@@ -54,6 +54,7 @@ func (r *ProtocolLXD) UpdateServer(server api.ServerPut, ETag string) error {
 	return nil
 }
 
+// Deprecated: Use CheckExtension instead.
 // HasExtension returns true if the server supports a given API extension.
```

Comment:
Those two lines need to be swapped. The function description always goes first.

Reply:
Good call. I remember once looking this up but didn't find any official resources defining where this should go. Is it more our standard for this repo? I'll swap the lines.

Reply:
That's more a general go thing: https://go.dev/blog/godoc